docs/source/contents/4_draft_mapping.rst

```diff
@@ -2403,7 +2403,7 @@ XPath
 
     :code:`mods/subject[not(@valueURI)]/topic[not(@valueURI)]` OR
 
-    :code:`mods/subject[not(@valueURI)]/name[not(valueURI)]/namePart[not(valueURI)]`
```

Comment:
Addresses: https://hyp.is/6epdVBgJEe2mQ1fCK5n5iA/utk-mods-to-rdf.readthedocs.io/en/latest/contents/4_draft_mapping.html